conda_build/skeletons/pypi.py

```diff
@@ -856,23 +805,109 @@ def get_package_metadata(package, d, data, output_dir, python_version, all_extra
                 print(license_name)
                 print()
                 license_name = input("What license string should I use? ")
+        elif noprompt:
+            license_name = "UNKNOWN"
         else:
-            if noprompt:
-                license_name = "UNKNOWN"
-            else:
-                license_name = input(("No license could be found for %s on " +
-                                    "PyPI or in the source. What license should I use? ") %
-                                package)
+            license_name = input(
+                "No license could be found for %s on PyPI or in the source. "
+                "What license should I use? " % package
+            )
     else:
         license_name = ' or '.join(licenses)
     # remove the word license from the license
-    clean_license_name = re.subn(r'(.*)\s+license', r'\1', license_name, flags=re.IGNORECASE)[0]
+    clean_license_name = re.subn(
+        r'(.*)\s+license', r'\1', license_name, flags=re.IGNORECASE
+    )[0]
     d['license'] = clean_license_name
-    d['license_family'] = guess_license_family(license_name, allowed_license_families)
+    d['license_family'] = guess_license_family(
+        license_name, allowed_license_families
+    )
     if 'new_hash_value' in pkginfo:
         d['digest'] = pkginfo['new_hash_value']
 
 
+def get_entry_points(pkginfo):
+    """Look at the entry_points and construct console_script and gui_scripts entry_points for conda
+    :param pkginfo:
+    :return dict:
+    """
+    entry_points = pkginfo.get('entry_points')
+    if not entry_points:
+        return {}
+
+    if isinstance(entry_points, str):
+        # makes sure it is left-shifted
+        newstr = "\n".join(x.strip() for x in entry_points.splitlines())
+        _config = configparser.ConfigParser()
+
+        try:
+            # TODO: Should we use read_file instead of readfp?
+            #  Because readfp is deprecated
+            _config.readfp(StringIO(newstr))
```

Comment:
Should we replace the ``readfp`` by ``read_file`` ?

Reply:
Maybe worth trying, but I'm not sure what the knock-on effects might be.  Give it a try, but don't work too hard on this little detail.

Reply:
Sure! I might open another PR for that to see if it will impact.
This concern is just because I saw that ``readfp`` will be removed in the future from ``ConfigParser``